react-app/src/library/types/consumed-headset-events.ts

```diff
@@ -23,7 +23,7 @@ type Events = {
     [HeadsetEvents.deviceRejectedCall]: RejectCallEventInfo;
     [HeadsetEvents.loggableEvent]: EventInfo;
     [HeadsetEvents.webHidPermissionRequested]: { callback: any };
-    [HeadsetEvents.deviceConnectionStatusChanged]: { currentVendor: VendorImplementation, isConnected: boolean, isConnecting: boolean};
+    [HeadsetEvents.deviceConnectionStatusChanged]: string;
```

Comment:
This should be typed somehow. Maybe something like this:
```
export type DeviceConnectionStatus = 'connecting' | 'connected' | 'notConnected' | 'noVendor';
```
or whatever we settled on.